tests/e2e/nodes_scaleup_scaledown.go

```diff
@@ -130,7 +130,11 @@ var _ = ginkgo.Describe("[csi-file-vanilla] [csi-block-vanilla-serialized] Nodes
 			statefulset.Spec.VolumeClaimTemplates[len(statefulset.Spec.VolumeClaimTemplates)-1].Spec.AccessModes[0] =
 				v1.ReadWriteMany
 		}
-
+		if windowsEnv {
```

Comment:
This is added in multiple files. We should handle this via adding utility function and call that function in each of these files